include/ord/Design.h

```diff
@@ -172,6 +172,7 @@ class Design
     Max
   };
   float getNetCap(odb::dbNet* net, sta::Corner* corner, MinMax minmax);
+  float getPortCap(odb::dbITerm* pin, sta::Corner* corner, MinMax minmax);
```

Comment:
I would rather see APIs that require sta objects moved to Timing.h so that Design is more about the netlist (obd).